forms-flow-submissions/docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
     volumes:
       - ./:/app:rw
     environment:
-      DATABASE_URL: ${SUBMISSION_API_DB_URL:-postgresql://postgres:changeme@forms-flow-submission-db:5432/submissionapi}
+      DATABASE_URL: ${DATABASE_URL:-postgresql://postgres:changeme@forms-flow-submission-db:5432/submissionapi}
```

Comment:
we need to update SUBMISSION_API_DB_URL to DATABASE_URL in sample.env too

Reply:
Done